README.adoc

```diff
@@ -3,33 +3,36 @@
 // 2. Update the issues link in Contributing section in the readme file
 // 3. Update the discussion link in config.yml file in .github/ISSUE_TEMPLATE directory
 
-= repo-template
+= Teragrep Java Configuration Wrapper for Typesafe Config
 
 // Add a short description of your project. Tell what your project does and what it's used for.
 
-This is a template repository for Teragrep organization.
+CNF-02 is a library that provides immutable configuration for Java projects mainly for the Typesafe Config. Typesafe Config will be converted to https://github.com/teragrep/cnf_01[CNF-01]'s Configuration interface, so it is wise to read through its' documentation as well. It has CNF-01 as a dependency, so it provides all the functionality that it has.
```

Comment:
"...so it is wise to read through *its'* documentation as well."

remove ' and use just *its*. It-pronominin omistusmuoto on aina its eikä siihen tarvita muita lisukkeita merkkaamaan sitä.

Reply:
Fixed